app/API/V1/Entities/User.php

```diff
@@ -229,6 +236,11 @@ public function getRoles():?Collection
         return $this->roles;
     }
 
+    public function hasRole()
+    {
+        return !!$this->getRoles();
```

Comment:
Shouldn't this check for a specific role? It needs to see if the user has the 'user' role specifically

Reply:
Changed